src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,22 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
+        return users;
+    }
```

Comment:
Not sure if we really need here such sort of getter. Generally, the idea of the task was to build `AuthenticationService` on top of `UserService`, hence `users` should not be accessible directly from `login` method.